downstairs/src/extent_inner_raw_v2.rs

```diff
@@ -0,0 +1,1058 @@
+// Copyright 2023 Oxide Computer Company
+use crate::{
+    cdt,
+    extent::{check_input, extent_path, ExtentInner, EXTENT_META_RAW_V2},
+    extent_inner_raw_common::{
+        pread_all, pwrite_all, OnDiskMeta, BLOCK_META_SIZE_BYTES,
+    },
+    mkdir_for_file,
+    region::JobOrReconciliationId,
+    Block, CrucibleError, ExtentReadRequest, ExtentReadResponse, ExtentWrite,
+    JobId, RegionDefinition,
+};
+
+use crucible_common::{BlockOffset, ExtentId};
+use crucible_protocol::ReadBlockContext;
+use slog::{error, Logger};
+
+use std::io::{BufReader, Read};
+use std::os::fd::{AsFd, AsRawFd};
+use std::path::Path;
+use std::{
+    fs::{File, OpenOptions},
+    io::IoSlice,
+};
+use zerocopy::AsBytes;
+
+pub(crate) const DEFAULT_ZFS_RECORDSIZE: u64 = 128 * 1024;
```

Comment:
both cases in `get_record_size` where this is used, it doesn't seem like the path is a ZFS file system, so maybe this should be `NON_ZFS_RECORDSIZE` or something?

Reply:
Good point, renamed to `DUMMY_RECORDSIZE` in f7c13d7d4369